plugins/shortcuts/src/AddShortcut.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { useLocation } from 'react-router';
+import { SubmitHandler } from 'react-hook-form';
+import { alertApiRef, useApi } from '@backstage/core';
+import {
+  Button,
```

Comment:
should probably consistently use the `@backstage/core` `Button` right?

Reply:
Hmm, then the `<Button/>` from `@backstage/core` will require me to add an `to` prop. As these buttons are not links I feel it makes more sense to keep them from `@material-ui/core`. Or am I missing something?